src/Quaternion.jl

```diff
@@ -1,3 +1,13 @@
+"""
+    Quaternion{T<:Real} <: Number
+
+Quaternion number type with real and imaginary part of type `T`.
+
+`QuaternionF16`, `QuaternionF32` and `QuaternionF64` are aliases for
+`Quaternion{Float16}`, `Quaternion{Float32}` and `Quaternion{Float64}` respectively.
```

Comment:
```suggestion
`Quaternion{Float16}`, `Quaternion{Float32}`, and `Quaternion{Float64}`, respectively.
```

Reply:
The docstring of `Complex` does not include that commas there.

https://github.com/JuliaLang/julia/blob/1ece3f64e093ba47bcd5e7a01193b8c9b05d441e/base/complex.jl#L8-L9

I have searched commas in JuliaLang/julia repository, and I found it seems better to put `,` before `respectively`, but I still don't understand why we need `,` before `and`.

Reply:
The comma before `and` in a list (the Oxford comma) is entirely optional, but it should be consistently used or not used. I prefer it because it never reduces clarity, and there are some sentences that are not clear without it.

Reply:
Ah, I see. thanks for the explanation! (and sorry for my English skill:sweat:)